client/file-browser/get-files.js

```diff
@@ -1,7 +1,10 @@
-import { promises as fsPromises } from 'fs'
+import fs, { promises as fsPromises } from 'fs'
 import path from 'path'
 
 export default async function readFolder(folderPath) {
+  if (!fs.existsSync(folderPath)) {
```

Comment:
Generally avoid `Sync` methods in `fs`, as they'll block the whole process until they return. Alternatives would be to use `fsPromises.access` or (probably better) to put a try/catch around the `fsPromises.readdir` call below and throw this error if `err.code === 'ENOENT'`

Reply:
👍, I handled it using `.catch`